checks.d/mysql.py

```diff
@@ -6,6 +6,7 @@
 
 # 3p
 import pymysql
+import psutil # permissions issues?
```

Comment:
you should add a try except ImportError here. psutil might not be available on source installs that don't have a compiler